doc/Variables.md

```diff
@@ -39,6 +48,14 @@ The CleverTap Flutter plugin provides several callbacks for the developer to rec
 - `onVariablesChanged`
 - `onValueChanged`
 
+From version `3.0.0` onwards, the following callbacks are also supported:
+
+- `onOneTimeVariablesChanged`
+- `onVariablesChangedAndNoDownloadsPending`
```

Comment:
We can probably add an example for these in the docs? @kushCT

Reply:
Will raise another PR for all doc changes @Anush-Shand